src/jesse_lib.erl

```diff
@@ -25,7 +25,6 @@
 
 %% API
 -export([ empty_if_not_found/1
-        , get_schema_id/1
```

Comment:
in some use cases we default from our fun to get_schema_id/1. see comment above about load_schemas arity